tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -264,6 +288,129 @@ public function test_is_merchant_enabled_returns_false_when_disabled() {
+	 * Test get_feed_hash returns empty string when file is missing or unreadable.
+	 *
+	 * @return void
+	 */
+	public function test_get_feed_hash_returns_empty_when_file_missing() {
+		$integration = new \WC_Stripe_Agentic_Commerce_Integration();
+		$get_hash    = new \ReflectionMethod( \WC_Stripe_Agentic_Commerce_Integration::class, 'get_feed_hash' );
+		$get_hash->setAccessible( true );
+
+		$this->assertSame( '', $get_hash->invoke( $integration, '' ) );
+		$this->assertSame( '', $get_hash->invoke( $integration, '/nonexistent/path/feed.csv' ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_feed_hash_returns_empty_when_file_missing is not named in camelCase. (undefined)

(CamelCaseMethodName)